src/SensioLabs/Connect/Api/Entity/Club.php

```diff
@@ -15,6 +15,7 @@
  * Club
  *
  * @author Marc Weistroff <marc.weistroff@sensiolabs.com>
+ * @deprecated since Connect 4.2.4, will be removed in 5.0.
```

Comment:
4.3 everywhere
let's also add runtime deprecation notices below the namespace, see how we do them in symfony

Reply:
Fixed.